src/plugins/irc-events/link.js

```diff
@@ -103,6 +102,8 @@ function fetch(url, cb) {
 	try {
 		var req = request.get({
 			url: url,
+			maxRedirects: 3,
+			timeout: 3000,
```

Comment:
Are these arbitrary? If so, maybe we could go a bit higher, like `5`/`10000`, but I don't have an opinion.
I guess we would empirically adjust these values, worst case.